Sources/Plasma/CoreLib/hsStream.cpp

```diff
@@ -410,6 +410,52 @@ bool hsStream::ReadLn(char *s, uint32_t maxLen, const char beginComment, const c
     return true;
 }
 
+ST::string hsStream::ReadLn(const char beginComment, const char endComment)
+{
+    ST::string result;
+
+    {
+        ST::string_stream ss;
+        char c;
+        char endCom = endComment;
+
+        while (true) {
+            while (!AtEnd() && strchr("\r\n", c = ReadByte()))
+                /* empty */;
+
+            if (AtEnd())
+                return result;
```

Comment:
This flavor of `ReadLn` has no way to distinguish between this EOF case and an empty line...  It would be better to pass the `ST::string` result by reference and still return a bool to distinguish between the two.